evap/results/exporters.py

```diff
@@ -60,15 +61,20 @@ def init_styles(self, workbook):
             palette_index = self.CUSTOM_COLOR_START + self.NUM_GRADE_COLORS + i
             style_name = self.deviation_to_style(deviation)
             self.create_style(workbook, deviation_base_style, style_name, palette_index, color)
+            self.create_style(workbook, deviation_base_style + ', left medium', style_name + '_total', palette_index, color)
```

Comment:
have you checked whether it's possible at all to have so many styles? at least on colors there is a surprisingly low maximum number.

also, i have the feeling that in create_style, there are now colors overwritten, please check that.